src/backend/aspen/api/views/auth.py

```diff
@@ -2,7 +2,7 @@
 from urllib.parse import urlencode
 
 from authlib.integrations.base_client.errors import OAuthError
-from authlib.integrations.starlette_client import StarletteRemoteApp
+from authlib.integrations.starlette_client import StarletteOAuth2App
```

Comment:
Any particular reason for the upgrade/change?

Reply:
I was trying to debug the `userinfo = token.get("userinfo")` line and turns out that syntax only works in newer versions of authlib 1.0.1 not our 0.15.4 version. I could make it work with 0.15.4 but I didn't seem like any more work to change the version number in our poetry file if everything else worked, which it did. So I thought, why not?

Reply:
Oh as for this particular change, it's because the `StarletteRemoteApp` class is gone in 1.0.1.

Reply:
👍